src/main/java/teammates/common/datatransfer/attributes/AccountAttributes.java

```diff
@@ -26,49 +27,95 @@
     public Date createdAt;
     public StudentProfileAttributes studentProfile;
 
-    public AccountAttributes(Account a) {
-        googleId = a.getGoogleId();
-        name = a.getName();
-        isInstructor = a.isInstructor();
-        email = a.getEmail();
-        institute = a.getInstitute();
-        createdAt = a.getCreatedAt();
-        studentProfile =
-                a.getStudentProfile() == null ? null : StudentProfileAttributes.valueOf(a.getStudentProfile());
+    AccountAttributes() {
```

Comment:
See if we can make it `private`?

Reply:
I tried, but it gave a PMD error. Even other PR's just had the default package level and not private

Reply:
Then NVM, we will follow the default package level.